migrations/2021-04-28-154439_denormalize_action_receipt_actions/up.sql

```diff
@@ -0,0 +1,27 @@
+ALTER TABLE action_receipt_actions
+    ADD COLUMN predecessor_account_id text NOT NULL DEFAULT '',
+    ADD COLUMN receiver_account_id text NOT NULL DEFAULT '',
+    ADD COLUMN included_in_block_timestamp numeric(20, 0) NOT NULL DEFAULT 0;
```

Comment:
We need indices for these columns

Reply:
It's also very desired to have:
- index on `args->>'method_name'`
- `base64_args` decoded into JSON field when possible, so that we can add indices on stuff like `receiver_id` for custom token transfers, etc

Reply:
oh wait, looks like it's already there 🤦 🤣